internal/jimm/ssh/ssh.go

```diff
@@ -10,10 +10,16 @@ import (
 
 	"github.com/canonical/jimm/v3/internal/dbmodel"
 	"github.com/canonical/jimm/v3/internal/errors"
+	"github.com/canonical/jimm/v3/internal/jimm/jujuauth"
 	"github.com/canonical/jimm/v3/internal/openfga"
 	"github.com/canonical/jimm/v3/internal/rpc"
 )
 
+type ConnInfo struct {
```

Comment:
godoc.. also, can we please call it `ConnectionInfo`.. or better `ControllerInfo`? omitting a few characters is saving nothing in the long run..